tools/cli/help.txt

```diff
@@ -313,8 +313,15 @@ Options:
                       (for example, --server=https://example.com:443).
   --architecture      Builds the server for a different architecture than your
                       developer machine's architecture. Valid architectures
-                      include os.osx.x86_64, os.linux.x86_64, and
-                      os.linux.x86_32.
+                      include os.osx.x86_64, os.linux.x86_64, os.linux.x86_32,
+                      and os.windows.x86_32. Note: This option selects the
+                      architecture matching version of binary-dependent
```

Comment:
This reads slightly odd to me/  Maybe just remove `matching version`?  Then I think it's good.